app/app/urls.py

```diff
@@ -71,7 +71,7 @@
     url(r'^issue/(?P<ghuser>.*)/(?P<ghrepo>.*)/(?P<ghissue>.*)', dashboard.views.bounty_details, name='issue_details_new2'),
     url(r'^bounty/details/?', dashboard.views.bounty_details, name='bounty_details'),
     url(r'^funding/details/?', dashboard.views.bounty_details, name='funding_details'),
-    url(r'^legacy/funding/details/?', dashboard.views.bounty_details, name='legacy_funding_details'),
```

Comment:
🤔

Reply:
huh, I could have sworn this was removed at some point on master, will put it back